src/common-components/tests/UnAuthOnlyRoute.test.jsx

```diff
@@ -96,7 +96,7 @@ describe('UnAuthOnlyRoute', () => {
     };
 
     delete window.location;
-    window.location = { href: getConfig().BASE_URL.concat(REGISTER_PAGE), search: '?variant=embedded' };
+    window.location = { href: getConfig().BASE_URL.concat(REGISTER_PAGE), search: `?variant=embedded&host=${DISCOVER_URL}` };
```

Comment:
Let's not use discover social url in tests and we don't really need a constant. Just use any dummy url i.e
```suggestion
    window.location = { href: getConfig().BASE_URL.concat(REGISTER_PAGE), search: '?variant=embedded&host=http://localhost/host-website' };
```